spec/docs/swagger_spec.rb

```diff
@@ -10,10 +10,57 @@
 
   context 'has valid paths' do
     it do
+      expect(subject).to validate(:get, '/manifests/{id}', 200, params)
+    end
+    it do
+      expect(subject).to validate(:get, '/manifests/{id}', 404, unknown_manifest_params)
+    end
+    it do
+      expect(subject).to validate(
+        # TODO must we require 'manifest' key? that is inconsistent with PATCH and /validate
```

Comment:
The `update` (PATCH) and `validate` methods do:
```
manifest_content = JSON.parse(request.body.read)
```

while the `create` (POST) method reads from params, depending on Rails to decode the JSON. This means a top-level param called `manifest`.

What that means is that the examples documentation for `create` are wrong, and inconsistent with the `validate` method (which IMO should have identical APIs except for the endpoint).

@jessieay thoughts on dropping the requirement for a top-level `manifest` param?